packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -34,9 +35,11 @@ interface ModelInstanceDocumentHeader extends ModelInstanceDocumentMetadata {
  */
 export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstanceDocument> {
   private readonly _schemaValidator: SchemaValidation
+  private readonly _maxContentLength: number | null
```

Comment:
this should probably be a hard-coded constant, but then for testing you can reach into ModelInstanceDocumentHandler and change the value of the constant.